eng/packages/http-client-csharp/generator/TestProjects/Local/Basic-TypeSpec/src/Generated/Internal/ClientPipelineExtensions.cs

```diff
@@ -14,11 +14,11 @@ namespace BasicTypeSpec
 {
     internal static partial class ClientPipelineExtensions
     {
-        public static async ValueTask<Response> ProcessMessageAsync(this HttpPipeline pipeline, HttpMessage message, RequestContext options)
+        public static async ValueTask<Response> ProcessMessageAsync(this HttpPipeline pipeline, HttpMessage message, RequestContext context)
         {
-            await pipeline.SendAsync(message, default).ConfigureAwait(false);
+            await pipeline.SendAsync(message, context.CancellationToken).ConfigureAwait(false);
```

Comment:
How do we know `context` is not null?

Reply:
Right.
Created https://github.com/Azure/azure-sdk-for-net/issues/46980 for follow-up since we need to add abstraction in MGC. Trying to limit the scope of current PR.

Reply:
Let's revert this change then and tackle the request context handling as part of the issue filed.

Reply:
reverted